### ISSUE TITLE
X-Permitted-Cross-Domain-Policies Header

### ISSUE THREAD

**Issue by manuel-sommer** [rfc-st/humble#8]
According to [OWASP Secure Headers Project](https://owasp.org/www-project-secure-headers/) , the HTTP Header X-Permitted-Cross-Domain-Policies Header should also be used. A check is mssing regarding this header.

**Comment by manuel-sommer** [comment]
@rfc-st I recommend you to add this project to hacktoberfest as you will then maybe receive also more PRs.

**Comment by rfc-st** [comment]
Hello!,

Thanks for your suggestion. I have included the check of this header in https://github.com/rfc-st/humble/commit/1bef54ca41cf6f1df0bb62e9421fe000e552c06c.

I cannot accept your PR for several reasons: it is not complete (this tool allows to show results in English and Spanish and therefore it is necessary to modify the corresponding files to show the messages in both languages). Additionally your PR removes an insecure value check from this header, which I think is necessary.

I have included your suggestion at https://github.com/rfc-st/humble/#acknowledgements. Thanks again for your time!.

Best regards,
